pkg/component/component.go

```diff
@@ -26,6 +26,7 @@ import (
 	"sync"
 	"syscall"
 
+	otrace "go.opentelemetry.io/otel/trace"
```

Comment:
Do you need the alias given that you don't import `pkg/trace` or anything like that ?